py/tests/unit/with_runtime_sparkling/test_deeplearning.py

```diff
@@ -45,3 +46,28 @@ def testLoadAndTrainMojo(prostateDataset):
     assert len(predMojo) == len(predModel)
     for i in range(0, len(predMojo)):
         assert predMojo[i] == predModel[i]
+
+
+def testInitialBiasAndWeightsAffectsTheResult(prostateDataset):
+    [traningDataset, testingDataset] = prostateDataset.randomSplit([0.9, 0.1], 1)
+    def createInitialDeepLearningDefinition():
+        return H2ODeepLearning(
+            seed=42,
+            reproducible=True,
+            labelCol="CAPSULE",
+            featuresCols=["AGE", "RACE", "DPROS", "DCAPS"],
+            hidden=[3, ])
+
+    referenceDeepLearning = createInitialDeepLearningDefinition()
+    referenceModel = referenceDeepLearning.fit(traningDataset)
+    referenceResult = referenceModel.transform(testingDataset)
+
+    deepLearning = createInitialDeepLearningDefinition()
+    matrix0 = [[.1, .2, .3, .4], [.4, .5, .6, .7], [.7, .8, .9, .6]]
+    matrix1 = [[.2, .3, .4], ]
+    deepLearning.setInitialWeights([matrix0, matrix1])
```

Comment:
In the current proposal, `initialWeights` takes an array of matrices (`Array[Array[Array[Double]]]]`) and `initialBiases` an array of vectors (`Array[Array[Double]]]`) I'm wondering whether it won't be confusing for users.

E.g.: 
`matrix0 = [[.1, .2, .3, .4], [.4, .5, .6, .7], [.7, .8, .9, .6]]` -> `[.1, .2, .3, .4]` is a row
`deepLearning.setInitialBiases([[.1, .2, .3], [.1]])` -> `[.1, .2, .3]` is a vector (column)

Reply:
Just an open idea. Would it make sense for this API to also expect Spark data frame with the data formatted as the matrices?

Reply:
> I'm wondering whether it won't be confusing for users

@mn-mikke what you could perhaps do is it use DenseMatrix from mllib, would that be an option? And DenseVector for biases.

Reply:
H2O type for `initial_weights` and `initial_biases` is array of H2OFrames. Each frame represents matrix or vector. Are you proposing to expose it as a list of dataframes?

I'm not sure what it's more user-friendly. To create a dataframe in Spark/Pyspark from scratch requires to write some boiler plate code and sometimes to specify the schema.

Reply:
I like the idea of using `DenseMatrix` and `DenseVector`. I hope there won't be any restriction for older versions of spark. @kuba Or is this, what you meant? WDYT?

Reply:
Yup, that would be perfect!

Just small note: I would try to use `ml` package as much as possible as `mllib` is considered  the "older one"

Reply:
@mn-mikke The different sizes between the weight and bias should not be confusing to users.  They are pretty standard.